includes/class-wp-statistics-meta-box.php

```diff
@@ -340,6 +333,21 @@ public static function getList($meta_box = false)
             ),
         );
 
+        // Load Upgrade metabox if user is not premium
```

Comment:
Btw, is this compatible with Add-on Customization? @foadadeli

Reply:
Should be 🤔 , since if users have valid Customization add-on license, they would be considered as premium, right?

Reply:
We probably don't need to show the "**Premium**" if one of the below options is enabled at least.

- Settings → Customization → White label
- Settings → Customization → Overview Widget Customization

Isn't it? @navidkashani